rust/src/lib/ifaces/base.rs

```diff
@@ -50,11 +50,10 @@ pub struct BaseInterface {
     /// every two characters. Case insensitive when applying.
     /// Serialize and deserialize to/from `mac-address`.
     pub mac_address: Option<String>,
-    #[serde(skip)]
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Since it is a query only property, how about adding `skip_deserializing` ?

Reply:
`skip_deserializing` will cause `nmstatectl fmt` lose it.